src/Bloc/BlSpace.class.st

```diff
@@ -1480,6 +1493,12 @@ BlSpace >> pulseRequested [
 	^ nextPulseRequested
 ]
 
+{ #category : #'startup - shutdown' }
+BlSpace >> rememberVisibleStatus [
+	
+	previousVisibleStatus := self isVisible
```

Comment:
I trust in your solution, but I didn't realize yet why isVisible is not enough, and we need a new flag

Reply:
Because we need to store the flag. Once the real window is destroyed / do not exist in the starting image, the window is always not visible. We need to know how it was before saving the image. Thanks for the review. I tried to avoid it but the space only delegates this behaviour to the real window.